NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -226,11 +232,15 @@
                                                'ggH_NNLOPS_Weight',
                                                'overallEventWeight',
                                                'Pileup_nTrueInt'
+                                               'LHEPart*'
```

Comment:
Also I suppose you have to add `LHEMela*` instead of LHEPart? (in cloneBranches and branchsel_out)